README.md

```diff
@@ -16,20 +20,35 @@ Follow the official documentation to setup an observer:
  - [mainnet](https://docs.elrond.com/validators/mainnet/config-scripts/)
  - [devnet](https://docs.elrond.com/validators/elrond-go-scripts/config-scripts/)
 
-Before starting the observer, make sure to edit the `config/prefs.toml` and set the target shard (for example, shard 0):
+Before starting the observer, make sure to edit the `config/prefs.toml`:
 
 ```
 [Preferences]
-   DestinationShardAsObserver = "0"
+   DestinationShardAsObserver = "0" # a chosen shard to observe
+   FullArchive = true
 ```
 
-Furthermore, enable the **database lookup extensions** in `config/config.toml`, as follows:
+Furthermore, adjust `config/config.toml`, as follows:
 
 ```
+[GeneralSettings]
+    StartInEpochEnabled = false
+
```

Comment:
add `...` between sections? one might think these lines are next to each other

Reply:
Added.